features/page-visibility.yml

```
@@ -0,0 +1,10 @@
+name: Page Visibility
+description: "The Page Visibility API tells you the current visibility state of the page, and fires events you can watch for to know when it becomes visible or hidden."
```

Naming the entry points here would be nice, as would explaining what visibility actually is. I also think we shouldn't mention the event, since we're pinning to caniuse, which doesn't acknowledge it.

```suggestion
description: "The `document.visibilityState` and `document.hidden` properties tell you whether the page is visible to the user (for example, it's not minimized or in a background tab)."
```